includes/Core/Modules/Module.php

```diff
@@ -816,4 +816,24 @@ function( $item ) {
 	public static function is_force_active() {
 		return false;
 	}
+
+	/**
+	 * Checks whether the module is shareable.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return bool True if module is shareable, false otherwise.
+	 */
+	public function is_shareable() {
+		if ( $this instanceof Module_With_Owner && $this->is_connected() ) {
+			$definitions = $this->get_datapoint_definitions();
+			foreach ( $definitions as $modules => $details ) {
+				if ( array_key_exists( 'shareable', $details ) && true === $details['shareable'] ) {
+					return true;
+				}
```

Comment:
How about something like this:
```suggestion
				return ! empty( $details['shareable'] );
```

Reply:
@eugene-manuilov The unit test for `test_get_shareable_modules` method fails for this approach.
```php
$shareable_active_modules = array_map( 'get_class', $modules->get_shareable_modules() );
```
Since `shareable_active_modules` returns an empty array in the test.

However, having an explicit if condition works 👍 
```php
if ( ! empty( $details['shareable'] ) ) {
	return true;
}
```

Reply:
Yes, my suggestion was wrong. Sorry, @hussain-t. :man_facepalming: